bin/main.ml

```diff
@@ -30,7 +31,9 @@ let set_common c =
   Clflags.debug_dep_path := c.debug_dep_path;
   Clflags.debug_findlib := c.debug_findlib;
   Clflags.dev_mode := c.dev_mode;
-  Printf.eprintf "Workspace root: %s\n" c.root;
+  Clflags.verbose := c.verbose;
+  if !Clflags.verbose then
+    Printf.eprintf "Workspace root: %s\n" c.root;
```

Comment:
Ok for the logging. However the chdir must stay here as it must be done before `Main.setup ...`